docs/src/content/concepts-modes.md

```diff
@@ -138,6 +139,62 @@ most cases, the configuration will look like this:
 
 {{< figure src="/schematics/proxy-modes-transparent-3.png" >}}
 
+## WireGuard (transparent proxy)
+
+```shell
+mitmdump --mode wireguard
+```
+
+The WireGuard mode works in the same way as transparent mode, except that setup
+and routing client traffic to mitmproxy are different. In this mode, mitmproxy
+runs an internal WireGuard server, which devices can be connected to by using
+standard WireGuard clients applications, and just by importing the WireGuard
+configuration provided by mitmproxy.
+
+No additional routing configuration is required for this mode, since WireGuard
+operates by exchanging UDP packets between client and server (with encrypted IP
+packets as their payload) instead of routing IP packets directly. Additionally,
+the WireGuard server runs entirely in userspace, so no administrative privileges
+are necessary for setup or operation of mitmproxy in this mode.
+
+### Configuration
+
+By default, the WireGuard server will listen on port `51820/udp`, the default
+port for WireGuard servers, which should not require modifying firewall settings
+in most circumstances. However, changing firewall settings might be necessary
+when specifying a different port (i.e. when using something like
+`--mode wireguard@51821`).
```

Comment:
```suggestion
port for WireGuard servers. This can be changed by setting the `listen_port` 
option or by specifying an explicit port (`--mode wireguard@51821`).
```

(let's avoid the firewall part here, I'm not sure if anyone is any wiser after reading that changing the firewall may or may not be necessary)

Reply:
Makes sense. I'll adapt.